meta-mender-clearfog/templates/local.conf.append

```diff
@@ -1,12 +1,32 @@
 # Appended fragment from meta-mender-community/meta-mender-clearfog/templates
 
+# Uncomment the lines below depending on the model
+
+# Clearfog - Base
 MACHINE ?= "clearfog-base"
+MENDER_IMAGE_BOOTLOADER_FILE = "u-boot-clearfog-base.bin"
+
+# Clearfog - Pro
+#MACHINE ?= "clearfog-pro"
+#MENDER_IMAGE_BOOTLOADER_FILE = "u-boot-clearfog-pro.bin"
+
+# Uncomment the line below which matches the storage medium you intend to use
+
+# SD Card
+UBOOT_BINARY ?= "u-boot-spl-sdhc.kwb"
+# SPI Interface
+#UBOOT_BINARY ?= "u-boot-spl-spi.kwb"
+# NAND
+#UBOOT_BINARY ?= "u-boot-spl-nand.kwb"
+# SATA
+#UBOOT_BINARY ?= "u-boot-spl-sata.kwb"
+# eMMC
+#UBOOT_BINARY ?= "u-boot-spl-mmc.kwb"
+# UART (Serial)
+#UBOOT_BINARY ?= "u-boot-spl-uart.kwb"
 
 IMAGE_INSTALL_append = " kernel-image kernel-devicetree"
 
 MENDER_FEATURES_ENABLE_append = " mender-uboot mender-image-sd"
 MENDER_FEATURES_DISABLE_append = " mender-grub mender-image-uefi"
 
-MENDER_IMAGE_BOOTLOADER_FILE = "u-boot-clearfog-base.bin"
-MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET = "1"
```

Comment:
You removed `MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET`, this is required for it to put the `MENDER_IMAGE_BOOTLOADER_FILE` at the correct location.

Is it something else then `1` on Clearfog Pro?

Reply:
I'm not sure when or why I removed it but it didn't end up causing an issue so I forgot about it.

I have restored the setting and tested with eMMC and there's no problems.